migrating/done/ossm-migrating-complete-assembly.adoc

```diff
@@ -2,6 +2,56 @@
 [id=ossm-migrating-complete-assembly.adoc]
 = Completing the Migration
 include::_attributes/common-attributes.adoc[]
-:context: ossm-migrating-deleting-2-6-assembly
+:context: ossm-migrating-complete-assembly
 
 toc::[]
+
+At this stage, you have completed the migration process. It is safe to remove {SMProduct} {smv2version}, complete the migration of the cert-manager tool, and setup network policies.
```

Comment:
This is a bit confusing. Migration of cert-manager and network policies should be already migrated. Let's remove "complete the migration of the cert-manager tool, and setup network policies."